deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1PodHandler.java

```diff
@@ -570,4 +566,33 @@ public boolean hasPodParent(ApiClient apiClient, String selector, String namespa
         }
 	    return podParent == null ? false : true;
 	}
+	
+    public List<PodParent> getParentsForAllNamespaces(ApiClient apiClient, String selector, boolean label)
```

Comment:
Why do we need it in Pod Handler?

Reply:
On lines with other getParent method.

Reply:
Why do we need getParentsForAllNamespaces method ? It does not fit anywhere on the contract of the class